lusSTR/cli/gui.py

```diff
@@ -0,0 +1,733 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2024, DHS.
+#
+# This file is part of lusSTR (http://github.com/bioforensics/lusSTR) and is licensed under
+# the BSD license: see LICENSE.txt.
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+#################################################################
+#              Importing Necessary Packages                     #
+#################################################################
+
+import json
+import importlib.resources
+import streamlit as st
+from streamlit_option_menu import option_menu
+import yaml
+import subprocess
+import os
+import re
+
+# ------ Packages For File/Folder Directory Selection --------- #
+
+import tkinter as tk
+from tkinter import filedialog
+
+
+
+# Create a global Tkinter root window
+try:
+    root = tk.Tk()
+    root.withdraw()  # Hide the root window
+except:
+    print("No GUI available!")
```

Comment:
We can drop this code now, right?

Reply:
This is what is allowing the GitHub CI to pass. Without, it fails.

Reply:
That's really surprising to me. The file_selector and folder_selector scripts that *actually* use tk don't touch this code.

Reply:
It seemed to be hung up on the ```from lusSTR.cli import gui``` line from the ```cli/__init.py__``` script.

Reply:
I understand that having `root = tk.Tk()` in there without a try/except block will cause issues on CI. What I'm suggesting is that we shouldn't need tk anywhere in gui.py now, since that's all been moved to accessory scripts where tk can be run in the main thread.

Reply:
Oooooooooh. Yes you are right!